src/benchmark.cc

```diff
@@ -110,6 +140,37 @@ namespace benchmark {
 
 namespace internal {
 
+// Performance measurements always come with random variances. Defines a
+// factor by which the required number of iterations is overestimated in order
+// to reduce the probability that the minimum time requirement will not be met.
+const double kSafetyMultiplier = 1.4;
+
+// Wraps --benchmark_min_time and returns valid default values if not supplied.
+double GetMinTime() {
+  const double min_time = FLAGS_benchmark_min_time;
+  if (min_time >= 0.0) {
+    return min_time;
+  }
+
+  if (FLAGS_benchmark_enable_random_interleaving) {
+    return kDefaultMinTimeTotal / kDefaultRepetitions;
+  }
+  return kDefaultMinTimeTotal;
+}
+
+// Wraps --benchmark_repetitions and return valid default value if not supplied.
+size_t GetRepetitions() {
+  const int repetitions = FLAGS_benchmark_repetitions;
```

Comment:
same here as above.. i'm surprised that random interleaving doesn't allow the repetition count to be overridden.

Reply:
same here, as long as repetitions >=0.0 I return repetitions, no matter whether enable is true or not.

Reply:
PTAL